src/GloriousProtonManager/gpm.py

```diff
@@ -132,7 +132,7 @@ def delete_old_release():
             print("No Proton-GE versions found on your system\n")
         else:
             list_installed_versions()
-    if event == "1. List past releases (Last 15)":
+    if event == "1. List last 15 releases":
```

Comment:
Please have in mind that the sg.buttons string text has to match 100% the events that are being called, so 
`if event == "1. List last 15 releases":`  the button should be fixed to `[sg.Button('1. List last 15 releases'), size=(47, 3), font=('DejaVu 9'), button_color=BUTTON_COLOR)],` also. Once that's corrected I can merge

Reply:
Done.